modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/facet/FacetBucketUtil.java

```diff
@@ -37,7 +38,9 @@ public static boolean isFieldInBucket(
 
 			String value = field.getValue();
```

Comment:
do you think `value` would also need a null check?

Reply:
Does our Field impl permits null values? 🤔 @BryanEngler

Reply:
i think so, but if the field didnt have a value, it probably wouldnt be in a facet right? so were probably fine

Reply:
I think so. But I can check tomorrow if you prefer to.